hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmSnapshotInfo.java

```diff
@@ -174,6 +177,8 @@ public void testSnapshotInfoProtoToSnapshotInfo() {
         snapshotInfoActual.getExclusiveSize());
     Assert.assertEquals(snapshotInfoExpected.getExclusiveReplicatedSize(),
         snapshotInfoActual.getExclusiveReplicatedSize());
+    Assert.assertEquals(snapshotInfoExpected.getExpandedDeletedDir(),
```

Comment:
Can we just `assertEquals` on `snapshotInfoExpected` and `snapshotInfoActual`?

Reply:
Yes we can, but this is just to check each field individually.

Reply:
`assertEquals()` will do the same thing as long as equals method is overridden.
https://github.com/junit-team/junit4/blob/main/src/main/java/org/junit/Assert.java#L145